source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,16 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.15.0:
+
+{+atlas-cli+} 1.15.0
+--------------------
+
+Released 22 February 2024
+
+- Fixes a bug that prevented you from using the ``--file`` flag with
```

Comment:
I think we need to callout the atlas search nodes that we added here. 

`- Adds atlas search  nodes commands`

Reply:
resolving in favour of ciprian's comment